### ISSUE TITLE
DeepL support

### ISSUE THREAD

**Issue by Vistaus** [naraesk/krunner-translator#20]
I really love this runner! However, I prefer to use DeepL, so it would be great if you could add support. :-)

**Comment by X-Amadeus-X** [comment]
Yeah, DeepL has really good results. I would also welcome an integration with krunner.

**Comment by FeWa-zu** [comment]
I agree!
Deepl is vastly superior in its translation, an integration would be great!

**Comment by wokawoka** [comment]
+1

**Comment by GermanZun** [comment]
Yes, DeepL would be great!

**Comment by shenghuang147** [comment]
It seems that this project has not been maintained for a long time, I don't know c++, but I tried to add Deepl support and it seems to be working properly, I hope someone can help me improve the code.
Here is the link to the code: [Add Deepl API](https://github.com/shenghuang147/krunner-translator/tree/add_deepl_api)

![图片](https://github.com/user-attachments/assets/eede02b0-4ec6-4895-9cc3-9c2769d69051)

![图片](https://github.com/user-attachments/assets/9f2a62a6-51b6-409c-bc11-ff2c688485e5)

![图片](https://github.com/user-attachments/assets/6bc7ee35-598d-4667-a9f8-f5db530262aa)

**Comment by naraesk** [comment]
Thanks, I'll have a look and add DeepL support.

**Comment by naraesk** [comment]
v2.1 finally supports DeepL.
